projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc-conflict-dialog-e2e-spec-flaky.ts

```diff
@@ -0,0 +1,84 @@
+import * as conflictDialog from '../../../helpers/product-configurator-conflict-dialog';
+import * as configuration from '../../../helpers/product-configurator';
+import * as configurationVc from '../../../helpers/product-configurator-vc';
+import * as configurationOvVc from '../../../helpers/product-configurator-overview-vc';
+import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
+
+const electronicsShop = 'electronics-spa';
+const testProduct = 'CONF_CAMERA_SL';
+
+const RB = 'radioGroup';
+
+context('Product Configuration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    configurationVc.registerConfigurationRoute();
+    configurationVc.registerConfigurationUpdateRoute();
+    configurationVc.goToConfigurationPage(electronicsShop, testProduct);
+    clickAllowAllFromBanner();
+  });
+
+  it('should follow the immediate conflict resolution process', () => {
+    configurationVc.clickOnNextBtnAndWait('Specification');
```

Comment:
@Uli-Tiger, it is not clear for what will be waiting. I would suggest `clickOnNextBtn` method name.

Reply:
there is already a clickOnNextBtn method within configuration-helper (non vc), which does not wait for the GET request to happen. This method waits for the GET request.
As this method was not introduced with this PR and as it used within most cypress tests, i would suggest not to rename it, as i it would enlarge the PR and complicate merging later.